low-level/framework/utils/iem.py

```diff
@@ -171,6 +171,5 @@ def generate_iem(module, event_code, severity, description):
                 os.remove(IEM_INIT_FAILED)
             EventMessage.send(module=module, event_id=event_code,
                               severity=severity, message_blob=description)
-        except EventMessageError as e:
-            logger.error("Failed to send IEM alert."
-                         f"Error:{e}")
+        except (EventMessageError, Exception) as e:
+            logger.error(f"Failed to send IEM alert. Error:{e}")
```

Comment:
shouldn't this IEM alert should get accumulated to be sent later !?

Reply:
Yes, done.